src/test/java/bio/terra/service/common/gcs/GcsUriUtilsTest.java

```diff
@@ -0,0 +1,104 @@
+package bio.terra.service.common.gcs;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Our code uses both `MatcherAssert.assertThat` and `Assertions.assertThat`. They have different calling forms but cover the same functionality, so it's confusing to use both. I think we should standardize on the MatcherAssert one, which is more commonly used. So it would be good to switch this code over as part of this PR.